### PR TITLE
feat(learn, i18n): enable japanese certifications

### PR DESCRIPTION
**Pull request by sidemt**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

I refered to the following PRs (when we enabled Ukrainian language)
https://github.com/freeCodeCamp/freeCodeCamp/pull/44454
https://github.com/freeCodeCamp/freeCodeCamp/pull/44461/

### CODE REVIEW

**Comment by sidemt** [comment]
@nhcarrigan Do we also need the configurations mentioned under "Enabling Localized Videos" section in the document below? I think we have not localized the video contents (like subtitles) for Japanese.
https://contribute.freecodecamp.org/#/how-to-test-translations-locally?id=enabling-localized-videos

Looks like there are only espanol, italian, portuguese in the current code.

**Comment by nhcarrigan** [comment]
We shouldn't need that configured until we have localised videos to display. 🙂

**Comment by sidemt** [comment]
@nhcarrigan Understood. Thank you 🙂

File: client/i18n/locales/japanese/links.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (nhcarrigan) [inline client/i18n/locales/japanese/links.json:1]: Any of the links in this file that have been translated to Japanese can be replaced with the appropriate URL

Comment (nhcarrigan) [inline client/i18n/locales/japanese/links.json:1]: Any of the links in this file that have been translated to Japanese can be replaced with the appropriate URL

File: client/i18n/locales/japanese/meta-tags.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (nhcarrigan) [inline client/i18n/locales/japanese/meta-tags.json:1]: These can also be translated - we can work with Quincy and Ahmad to identify good SEO terms as well.

File: client/i18n/locales/japanese/motivation.json [hunk 3]
```diff
@@ -0,0 +1,819 @@
+{
```
Comment (nhcarrigan) [inline client/i18n/locales/japanese/motivation.json:1]: When we go to translate these, we can actually replace them with phrases and quotes that are more culturally relevant - we don't have to do exact translations (which is why this isn't on Crowdin).

File: client/i18n/locales/japanese/trending.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
+{
```
Comment (nhcarrigan) [inline client/i18n/locales/japanese/trending.json:1]: As we expand our Japanese news platform, we'll be able to use Japanese links here as well.

**Review by naomi-lgbt — DISMISSED** [review]
Pulled this down and built it, looks good to me.

I've left a few notes about the JSON files, to explain the next steps for translations. This is completely fine to merge as is, though. 🙂

**Comment by sidemt** [comment]
Thanks @nhcarrigan ! Yes, I just copied the English files and forgot to update. I will look into them.

**Comment by nhcarrigan** [comment]
Definitely not something you have to do in this PR. Once we get a second set of eyes on this and it is merged, we can let the contributors know they can look at the quotes and whatnot.

File: client/i18n/locales/japanese/links.json (hunk 1 above)
Comment (sidemt) [reply client/i18n/locales/japanese/links.json:1]: I updated some links which already have Japanese version, and the /news site which we would soon be avaiable.

File: client/i18n/locales/japanese/meta-tags.json (hunk 2 above)
Comment (sidemt) [reply client/i18n/locales/japanese/meta-tags.json:1]: I translated the title and some keywords which are often written in Japanese.

**Review by naomi-lgbt — APPROVED** [review]
This should be good to go!

**Review by raisedadead — APPROVED** [review]
@nhcarrigan - I am wondering if you can mark the outstanding conversations as resolved and we can get this in.

**Comment by sidemt** [comment]
@raisedadead Sorry I could resolve them so I did.

**Comment by sidemt** [comment]
@raisedadead Oops, sorry!😂 I wasn't sure how we should work with the resolve conversation button

**Comment by nhcarrigan** [comment]
Yes, those were definitely not meant to be blocking but more to indicate the next steps. Apologies for the hold up. 🙂

**Pull request merged**